docs/src/devdocs/dev_install.md

```diff
@@ -1,74 +1,201 @@
+* Install Atom from https://atom.io/  
+  Drag .app into Applications folder  
+  Run
 
-The Julia packages can be installed by either cloning the repository or though
-the standard installation (`Pkg.add(PkgName)`) followed by checking out the master branch
-(`Pkg.checkout("PkgName")`. The following code will check out the relevant packages:
+Ok, you should have Atom and a `Julia>` Julia-prompt side-by-side.
 
-```julia
-Pkg.checkout("Atom")
-Pkg.checkout("Blink")
-Pkg.checkout("CodeTools")
-Pkg.checkout("DevTools")
-Pkg.checkout("Hiccup")
-Pkg.checkout("Juno")
-Pkg.checkout("LNR")
-Pkg.checkout("Media")
+`Julia> Pkg.status()` from Julia prompt should say no packages installed!  `Packages` menu drop down in Atom shouldn't have "Julia" yet.
+
+* Use Atom's `apm` package manager to install `atom-julia-client` (Julia IDE for Atom) which makes use of `ink` (toolkit for building IDEs in Atom):
+
+```shell
+apm install https://github.com/JunoLab/atom-ink           # ~/.atom/packages/ink
+apm install https://github.com/JunoLab/atom-julia-client  # ~/.atom/packages/julia-client
 ```
 
-### Bleeding Edge Features (Plotting, Debugging, etc.)
+You can observe this creates a local mirror of each GitHub repo.  
+(Note: You can sync all local repos with `Pkg.update()`
+
+If you now relaunch Atom you will see the `Packages` menu drop down now has a "Julia" entry.
+  `Packages -> Julia -> Open Console`
+  Try `1+1` and it will fail "Julia could not be started!"
+
+(Note that `Julia> Pkg.status()` still reports nothing).
+  
+(**NOTE**: you can actually skip this step because when you do `1+1` a couple of steps further down it will automatically perform this command!)
+
+* `Julia> Pkg.add("Atom")`
 
-In order to use the bleeding edge features, you may need to checkout master on
-the respective repositories. For example, for the latest plotting features, you
-may need to
+Now `Julia> Pkg.status()` will list a ton of stuff:
 
 ```julia
-Pkg.checkout("Plots")
+Julia> Pkg.status()
+1 required packages:
+ - Atom                          0.5.4
+43 additional packages:
+ - ASTInterpreter                0.0.4
+ - AbstractTrees                 0.0.4
+ - ArgParse                      0.4.0
+ - BinDeps                       0.4.5
+ - Blink                         0.4.3
+ - COFF                          0.0.2
+ - ...
+```
+
+NOTE:
+
+```shell
+ls ~/.julia/lib/v0.5/  
+Atom.ji		CodeTools.ji   Hiccup.ji	HttpParser.ji	Juno.ji		MacroTools.ji	Mustache.ji	SHA.ji
+BinDeps.ji	Codecs.ji	   Homebrew.ji	HttpServer.ji	LNR.ji		MbedTLS.ji	Mux.ji		URIParser.ji
+Blink.ji	Compat.ji	   HttpCommon.ji	JSON.ji		Lazy.ji		Media.ji	Reexport.ji	WebSockets.ji
+
+ls ~/.atom/packages
+README.md	ink		julia-client	language-julia
+```
+
+Now restarting Atom, doing `Packages -> Julia -> Open Console -> 1+1` it still fails with "Julia could not be started!"
+
+* `Atom -> Prefs -> Packages -> 'julia client' -> Settings -> Julia Path -> /Applications/Julia-0.5.app/Contents/Resources/julia/bin/julia`
```

Comment:
This depends on the operating system and is already covered in the installation part of the manual. Again, this should already be known.